### PR TITLE
Fix of the untranslated value for AutocompleteSelect. Now use: autoco…

### LINKED ISSUE #1106: [AutocompleteSelect] Add translate text for error message
# Current behaviour

The error message is not translated:
```javascript
    _handleAutocompleteBadInput = value => {
        this.setState({customError: `${value} is not a valid entry.`});
    };
```
https://github.com/KleeGroup/focus-components/blob/43ed4a403b525c5847dd923f9c3055db66a24d8f/src/components/input/autocomplete-select/field.js#L29

# Expected behaviour

Add a translation key for this message

## Versions

Focus-core: 0.13.0 & 0.14.1
Focus-components: 0.13.0 & 0.14.1

### PR DESCRIPTION
**Pull request by durandx**
### Patch

Fix of the untranslated value for AutocompleteSelect. Now use the translated string : 
```javascript
autocomplete: {
  error: {
  badinput: "La valeur __input__ n est pas valide"
  }
 }
```
> Fixes #1106

### CODE REVIEW

**Comment by pierr** [comment]
@durandx  this component does not have  `this.translate` method please use `focus-core/translate`

**Comment by durandx** [comment]
Tested on my machine. It was working fine ...

**Comment by Bernardstanislas** [comment]
That might be because of the global `i18n` set by the components as a legacy feature, so please use `focus-core/translate` even if it is working for now

**Comment by durandx** [comment]
You're right i forget to copy the translation behaviour modification before commit

File: src/components/input/autocomplete-select/field.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React, {Component, PropTypes} from 'react';
 import AutocompleteSelectEdit from './edit';
 import AutocompleteSelectConsult from './consult';
+import Translation from '../../../behaviours/translation';
```
Comment (pierr) [inline src/components/input/autocomplete-select/field.js]: `import translate from 'focus-core/translate'`

File: src/components/input/autocomplete-select/field.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 
+@Translation
```
Comment (pierr) [inline src/components/input/autocomplete-select/field.js]: remove this

File: src/components/input/autocomplete-select/field.js [hunk 3]
```diff
@@ -26,7 +28,7 @@ class AutocompleteSelectField extends Component {
     };
 
     _handleAutocompleteBadInput = value => {
-        this.setState({customError: `${value} is not a valid entry.`});
+        this.setState({customError: this.i18n('autocomplete.error.badinput', {input: value}) });
```
Comment (pierr) [inline src/components/input/autocomplete-select/field.js]: `translate(autocomplete.error.badinput', {input: value})`

**Comment by pierr** [comment]
@durandx I just comment your code on the commit to be sure we are on the same page.
What you did is not wrong, it is just not what we do now. We tend to avoid the use of `this` as components may be pure functions.

**Comment by durandx** [comment]
What was wrong with my PR ?

**Comment by pierr** [comment]
Sorry it has been fixed #1153 but what you did was fine .

The problem was that you click the merge button on github who add dirty commit to the graph history.

**Comment by pierr** [comment]
but tthanks @durandx

**Pull request closed without merging**